### PR TITLE
origin.spec: replace 3.x client RPMs

### PR DESCRIPTION
**Pull request by sosiouxme**
fixes https://bugzilla.redhat.com/show_bug.cgi?id=1702769

### CODE REVIEW

**Comment by vikaslaad** [comment]
@smarterclayton @deads2k or @bparees could someone please approve this PR ?

File: origin.spec [hunk 1]
```diff
@@ -113,6 +113,7 @@ Provides:       atomic-openshift-node
 %package clients
 Summary:        %{product_name} Client binaries for Linux
 Provides:       atomic-openshift-clients
+Obsoletes:      atomic-openshift-clients
```
Comment (bparees) [inline origin.spec]: per discussion on slack this looks wrong:
https://coreos.slack.com/archives/CB95J6R4N/p1557155759054800

File: origin.spec [hunk 2]
```diff
@@ -122,6 +123,7 @@ Requires:       bash-completion
 %package clients-redistributable
 Summary:        %{product_name} Client binaries for Linux, Mac OSX, and Windows
 Provides:       atomic-openshift-clients-redistributable
+Obsoletes:      atomic-openshift-clients-redistributable
```
Comment (bparees) [inline origin.spec]: presumably this too.

**Comment by tbielawa** [comment]
The old package used to be called `atomic-openshift-<FOO>` now it's just called `openshift-<FOO>`. This PR `obsoletes` the _old_ package names so that new users can have a seamless upgrade experience to switch to the new `openshift-<FOO>` packages.

**Comment by tbielawa** [comment]
re: https://jira.coreos.com/browse/ART-599

**Pull request merged**